### PR TITLE
#124 Adapt top level readme

### PR DESCRIPTION
**Pull request by RMeissnerCC**
Updating readme and .env.exampe

### CODE REVIEW

File: tests/resources/quality-matrix-replication-source-request.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "replication_source": {
       "terms": {
         "field": "properties.ccm:replicationsource.keyword",
-        "size": 10000
```
Comment (MRuecklCC) [inline tests/resources/quality-matrix-replication-source-request.json:48]: Oh, I guess those depend on the .env file used to run the tests.
I wonder how the pipeline worked before or after that change!?

File: src/app/api/collections/quality_matrix.py [hunk 2]
```diff
@@ -123,10 +123,10 @@ def collection_quality_matrix(collection: Tree) -> QualityMatrix:
         A(
             "terms",
             field="collections.nodeRef.id.keyword",
-            size=10000,
+            size=ELASTIC_TOTAL_SIZE,
```
Comment (MRuecklCC) [inline src/app/api/collections/quality_matrix.py:126]: :+1:

File: tests/resources/quality-matrix-replication-source-request.json (hunk 1 above)
Comment (MRuecklCC) [reply tests/resources/quality-matrix-replication-source-request.json:48]: Nevermind, found the other related changes that make that work :)

File: README.md [hunk 3]
```diff
@@ -1,26 +1,124 @@
...
+## Unit-Tests
+
+Unit-tests are placed [here](./tests).
+
+All tests make use of mocks and static (checked in) resources. Hence, they can run independently of any docker
+container or other environment constraints.
+
+## Branching Model
+
+This repository follows the [GitHub Flow](http://scottchacon.com/2011/08/31/github-flow.html) branching model, but adds
```
Comment (MRuecklCC) [inline README.md:44]: I would just reference Git-Flow (https://www.atlassian.com/de/git/tutorials/comparing-workflows/gitflow-workflow) then. Git-Flow inherently comes with the develop branch -  very much simplified: "GitHub-FLow = Git-Flow - develop branch"

Comment (RMeissnerCC) [reply README.md:44]: Done!

**Pull request merged**